force-app/main/default/classes/ServiceSessionService.cls

```diff
@@ -38,4 +41,26 @@ public with sharing class ServiceSessionService {
 
         return serviceSessions;
     }
+
+    public Set<String> getServiceSessionStatusBuckets(String serviceSessionStatus) {
```

Comment:
Following up from my comment above, I think this method should be renamed to something like "getServiceSessionStatusesForBuckets" and the argument being passed in should be something like "bucketName". While the returned values look to be Service Session status values, what is being passed in appears to be the name of the bucket, not the status (see line #46 directly below this)